backend/src/Controller/BaseController.php

```diff
@@ -44,6 +44,7 @@ public function __construct(SerializerInterface $serializer)
     const ERROR_UPDATE_CAPTAIN_ONGOING = ["captain is ongoing,You can not modify detail or destination or deliveryDate", "9217"];
     const ERROR_ORDER_HIDE = ["order hide", "9218"];
     const ERROR_ORDER_ALREADY_DELIVERED = ["order is already delivered", "9219"];
+    const CAPTAIN_RECEIVED_ORDER_FOR_THIS_STORE = ["order is already delivered", "9200"];
```

Comment:
@habibdevsy The phrase "order is already delivered" have to be updated later to refer to the reception of a previous order from the same store by the captain.